src/tera.rs

```diff
@@ -951,6 +951,27 @@ mod tests {
         assert_eq!(result, "Hello world");
     }
 
+    #[test]
+    fn test_render_map_with_dotted_keys() {
+        let mut my_tera = Tera::default();
+        my_tera
+            .add_raw_templates(vec![
+                ("dots", "{{ map[\"a.b.c\"] }}"),
```

Comment:
Use raw strings `r#"{{ map["a.b.c"] }}"#` so the example is copy/pastable

Reply:
Haha, it's funny you mention it, I did use raw strings originally but didn't see them used in any of the other examples, so I switched it up.

Switched back to raw strings :)